cookiecutter/extensions.py

```diff
@@ -18,3 +20,17 @@ def jsonify(obj):
             return json.dumps(obj, sort_keys=True, indent=4)
 
         environment.filters['jsonify'] = jsonify
+
+
+class RandomStringExtension(Extension):
+    """Jinja2 extension to create a random string."""
+
+    def __init__(self, environment):
+        super(RandomStringExtension, self).__init__(environment)
+
+        def random_ascii_string(length, punctuation=True):
```

Comment:
I do not like punctuation inclusion by default. 
I am always expect more problems from special characters.

Reply:
Ok 👍